hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveSyncClient.java

```diff
@@ -316,4 +353,11 @@ public void updateTableComments(String tableName, List<FieldSchema> fromMetastor
     }
   }
 
+  Table getTable(String tableName) {
+    try {
+      return client.getTable(databaseName, tableName);
+    } catch (TException e) {
+      throw new HoodieHiveSyncException(String.format("Database: %s, Table: %s  does not exist", databaseName, tableName), e);
+    }
+  }
```

Comment:
where do we use this method? other methods from this class just call client.getTable(). we should not introduce random helpers that lowers the code quality